tests/network/hotplug.go

```diff
@@ -239,7 +240,11 @@ var _ = SIGDescribe("nic-hotplug", func() {
 					},
 				))
 			// we need to clean up the MACs because KubeMacPool sets MACs for VM interfaces ...
-			Expect(cleanMACAddressesFromSpec(vm.Spec.Template.Spec.Domain.Devices.Interfaces)).To(
+			Eventually(func() []v1.Interface {
```

Comment:
Does this need to be in `Eventually` or is it enough if the above routine is?

Reply:
So let's convert this one to `Expect` to better reflect what is expected.

Reply:
@xpivarc done

Reply:
Thank you!